app/assets/stylesheets/application.scss

```diff
@@ -19,30 +19,77 @@
   background-color: rgba(127, 255, 212, 0.6);
   padding: 10px;
 }
-
 .searches-card {
   border: 1px solid black;
   padding: 20px;
   border-radius: 20px;
   width: 30%;
 }
-
-#search-index-body {
+.about-us-card {
+  border: 1px solid black;
+  padding: 20px;
+  padding-bottom: 140px;
+  min-height: 387px;
+  border-radius: 20px;
+  width: 100%;
+  transform: rotate(20deg);
+  background-color: bisque;
+  margin-right: 50px;
+}
+.about-us-card2 {
+  border: 1px solid black;
+  padding: 20px;
+  border-radius: 20px;
+  width: 100%;
+  transform: rotate(-20deg);
+  margin-left: 50px;
+  background-color: bisque;
+}
+.about-us-card3 {
+  border: 1px solid black;
+  padding: 20px;
+  margin-top: 25px !important;
+  border-radius: 20px;
+  min-width: 540px;
+  max-width: 540px;
+  margin: 0 auto;
+  background-color: bisque;
```

Comment:
It looks like there's some shared styles between these. Could you combine the shared styles into a single class and then use ids for the things that are specific to each of the individual cards?

[BLOCKER]

Reply:
This was changed accordingly, thank you!